specification/web/resource-manager/Microsoft.Web/stable/2024-04-01/WebApps.json

```diff
@@ -6154,6 +6154,52 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/machinekey": {
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Updates the machine key of an app.",
+        "operationId": "WebApps_UpdateMachineKey",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the app.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
```

Comment:
PLease add a schema if you are returning something in the body

Reply:
waiting a response for this comment

Reply:
@razvanbadea-msft we are not returning anything in the body here as I commented above